### PR TITLE
Post/uncertainty in rs

### PR DESCRIPTION
**Pull request by bharathi-srini**

### CODE REVIEW

File: _posts/2019-10-01-uncertainty_in_RS.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+Neural networks are popular in the industry as they are extremely effective in learning complex patterns and capturing non-linearities. This would enable us to personalize search results using a learning framework that can infer travel preferences of our customers. There's also an added advantage of creating latent representations for the user when Embeddings are used for denoting users. Embeddings are trained when the neural network is optimized and this also gives a vector representation of users which can be reused for other machine learning problems where we want to identify similar users. However, traditional neural networks do not quantify the inherent uncertainty in the data or in the model predictions. This is something valuable while ranking and can be modeled by using a Bayesian approach.
+
+While Bayesian methods is a vast topic by itself, it will not be covered in detail here but a brief overview is presented in the section below if the reader is not yet acquainted with Bayesian Neural Networks.
```
Comment (mattilyra) [inline _posts/2019-10-01-uncertainty_in_RS.md]: How this is relevant for the article?

File: _posts/2019-10-01-uncertainty_in_RS.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Neural networks are popular in the industry as they are extremely effective in learning complex patterns and capturing non-linearities. This would enable us to personalize search results using a learning framework that can infer travel preferences of our customers. There's also an added advantage of creating latent representations for the user when Embeddings are used for denoting users. Embeddings are trained when the neural network is optimized and this also gives a vector representation of users which can be reused for other machine learning problems where we want to identify similar users. However, traditional neural networks do not quantify the inherent uncertainty in the data or in the model predictions. This is something valuable while ranking and can be modeled by using a Bayesian approach.
```
Comment (mattilyra) [inline _posts/2019-10-01-uncertainty_in_RS.md]: - we don't personalise search results, we personalise offers

- are the embeddings relevant for the rest of the article?

- you say that "This is something valuable while ranking and can be modeled by using a Bayesian approach." without explaining how or why it is valuable.

Comment (bharathi-srini) [reply _posts/2019-10-01-uncertainty_in_RS.md]: removed the sentences talking about embeddings.

File: _posts/2019-10-01-uncertainty_in_RS.md (hunk 1 above)
Comment (bharathi-srini) [reply _posts/2019-10-01-uncertainty_in_RS.md]: removed the justification of using NN

File: _posts/2019-10-01-uncertainty_in_RS.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+If a customer requests a round trip on the same day without explicitly mentioning the desired time of travel, it is reasonable to propose travel options for the outbound trip for hours in the morning and return flights departing in the evening. While such cases are handled by rule based filtering, it quickly gets exhausting to cover all possible cases. Therefore, it is useful to have a learning agent which could infer such patterns and does the selection of 'best' travel options for each customer.
+Step in, machine learning to save the day!
```
[the remark this answers is not in the document]
Comment (bharathi-srini) [reply _posts/2019-10-01-uncertainty_in_RS.md]: restructured it so that my grandma can understand it :D

**Pull request merged**